tests/serve_app/oxo_test.py

```diff
@@ -2243,6 +2292,81 @@ def testRunScanMutation_whenNetworkAsset_shouldRunScan(
     assert args["assets"][1].mask == "24"
 
 
+def testRunScanMutation_whenDomainAsset_shouldRunScan(
+    authenticated_flask_client: testing.FlaskClient,
+    agent_group_nmap: models.AgentGroup,
+    domain_asset: models.DomainAsset,
+    scan: models.Scan,
+    mocker: plugin.MockerFixture,
+) -> None:
+    """Test RunScanMutation for Domain asset."""
+    mocker.patch(
+        "ostorlab.cli.docker_requirements_checker.is_docker_installed",
+        return_value=True,
+    )
+    mocker.patch(
+        "ostorlab.cli.docker_requirements_checker.is_docker_working", return_value=True
+    )
+    mocker.patch(
+        "ostorlab.cli.docker_requirements_checker.is_swarm_initialized",
+        return_value=True
+    )
+    mocker.patch("docker.from_env")
+    mocker.patch(
+        "ostorlab.runtimes.local.runtime.LocalRuntime.can_run", return_value=True
+    )
+    scan_mock = mocker.patch(
+        "ostorlab.runtimes.local.runtime.LocalRuntime.scan", return_value=scan
+    )
```

Comment:
I think I've seen this as a fixture before, can you please double check..

Reply:
applied